KIPs/kip-81.md

```diff
@@ -428,50 +284,71 @@ A proposal contains textual descriptions and optionally on-chain transactions. T
 
 While GCs hold their voting rights from staked KLAYs, a special secretariat account manages the overall on-chain governance process. Governance proposals can be submitted by the secretariat account, or any GC if the secretariat account is not set. The secretariat account can be updated by on-chain governance.
 
-#### Voting steps
+#### Voting steps overview
 
 Under the default timing settings, a typical governance proposal is handled as follows.
 
 1. When a governance proposal is submitted, it enters a 7-day preparation period where GCs can adjust their voting powers by staking or unstaking KLAYs. The list of voters (i.e., GCs) is finalized at the moment of proposal submission. However, voting powers are finalized at the end of the preparation period.
 2. A 7-day voting period immediately follows after the preparation period.
-3. If there are enough Yes votes, the transactions can be queued within 7 days after voting ends.
+3. If there are enough Yes votes, the transactions can be queued within 14 days after voting ends.
```

Comment:
Don't we need to update the diagram from 7 days to 14 days?

Reply:
Fixed by [6016371](https://github.com/klaytn/kips/pull/105/commits/6016371c69a5bb45a4de4537f7ce25a4cf58242d).